### ISSUE TITLE
Can not compile

### ISSUE THREAD

**Issue by ghost** [drobilla/ingen#2]
Hello,
I have an error at compilation, I don't know where to look to fix it 

```
Build failed
 -> task in 'libingen' failed with exit status 1: 
        {task 139956757056200: cxx URI.cpp -> URI.cpp.1.o}

['/usr/bin/g++', '-I/tmp/ingen', '-DNDEBUG', '-Wlogical-op', '-Wsuggest-attribute=noreturn', '-Wunsafe-loop-optimizations', '-fshow-column', '-std=c++11', '-pthread', '-fPIC', '-I.', '-I..', '-I/usr/include/lilv-0', '-I/usr/include/sratom-0', '-I/usr/include/sord-0', '-I/usr/include/serd-0', '-I/tmp/ingen/build', '-DHAVE_GCOV=1', '-DHAVE_BOOST_FORMAT_HPP=1', '-DHAVE_BOOST_INTRUSIVE_SLIST_HPP=1', '-DHAVE_BOOST_INTRUSIVE_PTR_HPP=1', '-DHAVE_BOOST_OPTIONAL_HPP=1', '-DHAVE_BOOST_VARIANT_HPP=1', '-DINGEN_HAVE_THREAD_LOCAL=1', '-DHAVE_LV2=1', '-DHAVE_LILV=1', '-DHAVE_SUIL=1', '-DHAVE_SRATOM=1', '-DHAVE_RAUL=1', '-DHAVE_SERD=1', '-DHAVE_SORD=1', '-DHAVE_PORTAUDIO=1', '-DHAVE_POSIX_MEMALIGN=1', '-DHAVE_ISATTY=1', '-DHAVE_VASPRINTF=1', '-DHAVE_LIBDL=1', '-DHAVE_SOCKET=1', '-DPYTHONDIR="/usr/local/lib/python3.6/site-packages"', '-DPYTHONARCHDIR="/usr/local/lib/python3.6/site-packages"', '-DINGEN_BUILD_LV2=1', '-DHAVE_JACK=1', '-DHAVE_JACK_METADATA=1', '-DHAVE_JACK_PORT_RENAME=1', '-DINGEN_JACK_SESSION=1', '-DINGEN_SHARED=1', '-DINGEN_VERSION="0.5.1"', '-DINGEN_BUILD_CLIENT=1', '-DHAVE_GLIBMM=1', '-DHAVE_GTHREAD=1', '-DHAVE_GTKMM=1', '-DHAVE_NEW_GTKMM=1', '-DHAVE_GANV=1', '-DINGEN_BUILD_GUI=1', '-DHAVE_JACK_MIDI=1', '-DINGEN_DATA_DIR="/usr/local/share/ingen"', '-DINGEN_MODULE_DIR="/usr/local/lib"', '-DINGEN_BUNDLE_DIR="/usr/local/lib/lv2/ingen.lv2"', '../src/URI.cpp', '-c', '-o/tmp/ingen/build/src/URI.cpp.1.o']

 -> task in 'libingen' failed with exit status 1: 
        {task 139956757056424: cxx URIs.cpp -> URIs.cpp.1.o}

['/usr/bin/g++', '-I/tmp/ingen', '-DNDEBUG', '-Wlogical-op', '-Wsuggest-attribute=noreturn', '-Wunsafe-loop-optimizations', '-fshow-column', '-std=c++11', '-pthread', '-fPIC', '-I.', '-I..', '-I/usr/include/lilv-0', '-I/usr/include/sratom-0', '-I/usr/include/sord-0', '-I/usr/include/serd-0', '-I/tmp/ingen/build', '-DHAVE_GCOV=1', '-DHAVE_BOOST_FORMAT_HPP=1', '-DHAVE_BOOST_INTRUSIVE_SLIST_HPP=1', '-DHAVE_BOOST_INTRUSIVE_PTR_HPP=1', '-DHAVE_BOOST_OPTIONAL_HPP=1', '-DHAVE_BOOST_VARIANT_HPP=1', '-DINGEN_HAVE_THREAD_LOCAL=1', '-DHAVE_LV2=1', '-DHAVE_LILV=1', '-DHAVE_SUIL=1', '-DHAVE_SRATOM=1', '-DHAVE_RAUL=1', '-DHAVE_SERD=1', '-DHAVE_SORD=1', '-DHAVE_PORTAUDIO=1', '-DHAVE_POSIX_MEMALIGN=1', '-DHAVE_ISATTY=1', '-DHAVE_VASPRINTF=1', '-DHAVE_LIBDL=1', '-DHAVE_SOCKET=1', '-DPYTHONDIR="/usr/local/lib/python3.6/site-packages"', '-DPYTHONARCHDIR="/usr/local/lib/python3.6/site-packages"', '-DINGEN_BUILD_LV2=1', '-DHAVE_JACK=1', '-DHAVE_JACK_METADATA=1', '-DHAVE_JACK_PORT_RENAME=1', '-DINGEN_JACK_SESSION=1', '-DINGEN_SHARED=1', '-DINGEN_VERSION="0.5.1"', '-DINGEN_BUILD_CLIENT=1', '-DHAVE_GLIBMM=1', '-DHAVE_GTHREAD=1', '-DHAVE_GTKMM=1', '-DHAVE_NEW_GTKMM=1', '-DHAVE_GANV=1', '-DINGEN_BUILD_GUI=1', '-DHAVE_JACK_MIDI=1', '-DINGEN_DATA_DIR="/usr/local/share/ingen"', '-DINGEN_MODULE_DIR="/usr/local/lib"', '-DINGEN_BUNDLE_DIR="/usr/local/lib/lv2/ingen.lv2"', '../src/URIs.cpp', '-c', '-o/tmp/ingen/build/src/URIs.cpp.1.o']
```

**Comment by drobilla** [comment]
Please include the build log, the above doesn't say anything useful.  Since it's in URI.cpp, though, I imagine you need to use LV2 git like https://github.com/drobilla/ingen/issues/4

**Comment by drobilla** [comment]
Closing for lack of information, but probably fixed by a758bd3a anyway.
